_posts/en/2020-12-04-better-tech-through-micro-procurementpt2.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Better tech through micro-procurement: part two"
+ref: micro-procurement-p2
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team, and procurement experts at ESDC"
+date: "2020-12-04"
+---
+<!--markdownlint-disable MD033-->
+This is part two of a two part series looking at micro-procurement for digital work.
+In this second post, we look at what makes implementing micro-procurement challenging, as well as a sneak peek at what we are planning at ESDC.
+
+***
+
+## What’s stopping the GC from doing micro-procurement of digital work
+
+The web of procurement rules is real and we are lucky to have procurement officers who have had extensive training to help us navigate that web.
+They help ensure that things are done by the book and no-one loses their financial signing authority or worse, gets fired. Like all policy though, procurement policy is open to interpretation and because of this, appetite for risk (by that procurement officer or procurement team) becomes a factor in implementing any procurement innovation. It can be seen as safer to continue doing things the same way.
+
+Another challenge can be that while policy provides flexibility to allow for simplified micro-procurement, departmental rules around procurement are often hard-coded into large financial systems that are costly, or difficult to make changes to.
+And even when departmental rules are not hard-coded into systems, they can often be inflexible and hard to change. If you have heard "This is how we do this at x department" then you know what we mean.
+As a result, across the GC, smaller, low risk procurements usually go through the same process as much larger, complex procurements.
+
+In order for micro-procurement to be possible, we (IT) must break our work out into small chunks.
+We aren't used to doing this. We have been grouping tasks together into large projects for many years.  We do that for many reasons:
+
+- we follow a project management process with extensive governance which intentionally or unintentionally encourages grouping tasks together into one large project
+- the procurement process is unclear to many in IT and as a result often seen as painful. Employees seek to reduce the pain by going through the process fewer times with larger projects.
+- we have a history of working in a waterfall way rather than an agile way
```

Comment:
Good point! I could add a bullet about 'fear inadvertently contract splitting'. I think contract splitting (what it is and how we are mitigating that risk) could be a whole blog post. Probably would be worth doing that sooner rather than later as that is the first thing anyone thinks of when they hear about micro-procurement.

Reply:
Very good point Guillaume, although I'm not sure contract splitting is top of mind for IT people (although it certainly is for procurement staff).

I think too that contract splitting deserves its own blog post. We can then point directly to it whenever the question comes up :)